PowerEditor/src/MISC/Process/ProcessAvecThread/Process.h

```diff
@@ -29,6 +29,7 @@
 #pragma once
 
 #include <windows.h>
+#include "Common.h"

```

Comment:
Please see the commit message:
```
process: add missing include Common.h (generic_string)
```
This include is required for the typedef `generic_string`.